src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3937,7 +3937,8 @@ internal void UpdateTypes(ExecutionContext context, bool updateOnly)
 
             if (errors.Count > 0)
             {
-                var allErrors = new StringBuilder('\n');
+                // Note: don't use '\n' ('char' convert implicitly to 'ushort' and 'int' and initialize 'StringBuilder' size)
```

Comment:
What is the preferred solution do you see here? Remove the comment?
Complete solution I believe should be in the compiler.
Also we can do here:
```powershell
var allErrors = new StringBuilder(); 
allErrors.Append('\n'); 
```